files/cache_instance_list.rb

```diff
@@ -33,12 +40,14 @@ def write_instance_cache(instance_list)
 
   def ec2
     @ec2 ||= begin
-      Fog::Compute.new({
-        :provider => 'AWS',
-        :aws_access_key_id      => config[:aws_access_key],
-        :aws_secret_access_key  => config[:aws_secret_access_key],
-        :region                 => config[:aws_region]
-      })
+      opts = { :provider => 'AWS', :region => config[:aws_region] }
+      if config[:credential_file]
+        Fog.credentials_path = config[:credential_file]
```

Comment:
Do we need this? 
Per the docs we can just drop in a .fog file in the default location (home dir)
http://fog.io/about/getting_started.html

When we can just change the required from false to true, and that is it?